### PR TITLE
[DOCS] Class reference typos and fixes.

### PR DESCRIPTION
**Pull request by cbscribe**
Fixed description wording on several member variables.

Also corrected a reference in Timer - setting `wait_time` isn't a valid solution when you wish to set `time_left`.

### CODE REVIEW

File: doc/classes/DynamicFontData.xml [hunk 1]
```diff
@@ -12,13 +12,13 @@
 	</methods>
 	<members>
 		<member name="antialiased" type="bool" setter="set_antialiased" getter="is_antialiased" default="true">
-			Controls whether the font should be rendered with anti-aliasing.
+			If [code]true[/code], the font is rendered with anti-aliasing. Default value: [code]true[/code].
```
Comment (akien-mga) [inline doc/classes/DynamicFontData.xml]: The default value is already documented automatically (see `default="true"` on previous line), so it should no longer be mentioned manually.

File: doc/classes/SphereMesh.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
 			Full height of the sphere.
 		</member>
 		<member name="is_hemisphere" type="bool" setter="set_is_hemisphere" getter="get_is_hemisphere" default="false">
-			Determines whether a full sphere or a hemisphere is created.
+			If [code]true[/code], a hemisphere is created rather than a full sphere. Default value: [code]false[/code].
```
Comment (akien-mga) [inline doc/classes/SphereMesh.xml]: Same here.

File: doc/classes/DynamicFontData.xml (hunk 1 above)
Comment (cbscribe) [reply doc/classes/DynamicFontData.xml]: There are a lot of these scattered through the docs, I suppose we should remove them all.

Comment (akien-mga) [reply doc/classes/DynamicFontData.xml]: I already did this half a year ago when we added support for extracting default values, there shouldn't be (m)any left: #30184.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**